lux/extensions/smtp/views.py

```diff
@@ -33,12 +29,16 @@ def post(self, request):
         form = ContactForm(request, data=data)
         if form.is_valid():
             email = request.app.email_backend
-            subject = 'Message from %s' % request.config['APP_NAME']
-            email.send_mail(to=form.cleaned_data['email'],
-                            subject=subject,
-                            message=form.cleaned_data['body'])
-            data = dict(success=True,
-                        message="Message sent")
```

Comment:
The code as it is now will send any arbitrary list of emails that the app defines. I've already got both emails you mentioned here: https://github.com/bmlltech/bmll-portal/blob/issue222-enquiry-email/bmllportal/settings.py#L80